includes/class-amp-theme-support.php

```diff
@@ -119,6 +119,12 @@ public static function finish_init() {
 			self::register_paired_hooks();
 		}
 
+		// Enqueue AMP runtime.
+		wp_enqueue_script( 'amp-runtime' );
+
+		// Enqueue default styles expected by sanitizer.
+		wp_enqueue_style( 'amp-default', amp_get_asset_url( 'css/amp-default.css' ) );
```

Comment:
It looks like this is just moved from [enqueue_amp_default_styles()](https://github.com/Automattic/amp-wp/pull/1013/files#diff-2585472f207548f626a43a7ff3cab922L754), which only passed 2 arguments to `wp_enqueue_style()`. But could this use a `$version` as the 4th argument: `AMP__VERSION`?

Reply:
Oh good point! Yes,  this is needed.